metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorDatabaseService.java

```diff
@@ -42,7 +44,11 @@
  * @author zhenl
  */
 public class HiveConnectorDatabaseService implements ConnectorDatabaseService {
+    @Getter
+    @Setter
```

Comment:
Why is there a setter here? On final fields you won't ever be able to change them after they're initially set.